test/ttTransactionTestFiller.json

```diff
@@ -599,4 +599,18 @@
             "value": ""
         }
     }
+
+	"dataTx_bcValidBlockTest": {
+        "transaction": {
+            "nonce": "0",
+            "gasPrice": "50",
+            "gasLimit": "50000",
+            "to": "",
+            "data": "0x60056013565b6101918061001d6000396000f35b3360008190555056006001600060e060020a6000350480630a874df61461003a57806341c0e1b514610058578063a02b161e14610066578063dbbdf0831461007757005b610045600435610149565b80600160a060020a031660005260206000f35b610060610161565b60006000f35b6100716004356100d4565b60006000f35b61008560043560243561008b565b60006000f35b600054600160a060020a031632600160a060020a031614156100ac576100b1565b6100d0565b8060018360005260205260406000208190555081600060005260206000a15b5050565b600054600160a060020a031633600160a060020a031614158015610118575033600160a060020a0316600182600052602052604060002054600160a060020a031614155b61012157610126565b610146565b600060018260005260205260406000208190555080600060005260206000a15b50565b60006001826000526020526040600020549050919050565b600054600160a060020a031633600160a060020a0316146101815761018f565b600054600160a060020a0316ff5b56",
```

Comment:
ttTransactionTest does not execute transaction like in state tests. this code will not be executed. 
it would just try to create a valid Transaction instance from this fields. 
Perhaps it is better to move this test to stInitCodeTestFiller?